packages/extensions/core/lib/pipeline/plugin-endpoint.ts

```diff
@@ -107,7 +107,7 @@ export class AutoRestExtension extends EventEmitter {
       }
     });
     const writerProxy = new Writable({
-      write: (chunk: string | Buffer, encoding: string, callback: Function) => {
+      write: (chunk: string | Buffer, encoding: BufferEncoding, callback: Function) => {
```

Comment:
Was this needed for the tool somehow?

Reply:
I had to upgrade typescript and rush require consistency in the typescript version. This is just a type change and `BufferEncoding` is an union of string type(`"utf-8" | "utf16" ...`)

Reply:
Makes sense!